Request aggregations from Caesar

rogerhutchings opened the pull request:
- Removes the old aggregation request to the Zoo API
- Add the graphql-request library
- Request aggregation data from the Caesar aggregations endpoint, and
transform the returned data.

Fix #344

cc @VVH

marten commented on app/modules/transcribe/aggregations/aggregations.factory.js:
```diff
@@ -1,15 +1,28 @@
+    var _client = new GraphQLClient(appConfig.graphqlEndpoint);
+    var _query = `query Aggregation($workflowId: ID!, $subjectId: ID!) {
+        workflow(id: $workflowId) {
+            reductions(subjectId: $subjectId) {
```
Looks like you're only interested in the points+text clustering? In that case you should specify the reducer key you want: `reductions(subjectId: 1276468, reducerKey: "poly_line_text")`. Order of reductions is not guaranteed, so this will also help with https://github.com/zooniverse/shakespeares_world/pull/345/files#diff-b3ba4f0c9757d9c9f1c449df29447795R64 where you might sometimes get the results from the wrong reducer otherwise. (You'll need to change that to just be `[0]` or `first()`)

rogerhutchings replied:
thanks 👍

rogerhutchings commented:
After talking to @CKrawczyk and @VVH, I've fixed the aggregations filter, reduced the required consensus score to 2.5, and added a note about aggregations to the readme.

simoneduca commented on app/modules/transcribe/aggregations/aggregations.factory.js:
```diff
@@ -35,38 +52,47 @@ function AggregationsFactory($q, SubjectsFactory, zooAPI, zooAPIConfig, zooAPIPr
+            });
+    }
+
+    function _filterAggregations(aggregations) {
```
This is not called anywhere?

simoneduca replied:
It was missing in the promise chain.

simoneduca commented:
LGTM. I can see aggregation on subject and the code is clear and simple.
